### ISSUE TITLE
Approximately when should we see bounding boxes?

### ISSUE THREAD

**Issue by dicroce** [thtrieu/darkflow#431]
I am training using yolo-voc.cfg and the VOC2007 training dataset. I am currently at step 11707 and have been training for about 4 hours on an NVIDIA K80 GPU. My loss is down to 7.**** and it looks like it bouncing around there... I just went through the exercise of stopping the training and running a command like the following to test:

flow --imgdir /PATH/TO/TEST/JPEGS --model cfg/yolo-voc.cfg --load -1 --gpu 1.0

This created an output directory called "out" in my jpegs folders but unfortunately none of the images in there had bounding boxes... I somewhat expected this as it seems like people are having some trouble getting this to work... I have seen past answers say that the network needs to converge... Approximately how many steps to train VOC2007 to convergence? I can let this go, but I'd appreciate some kind of ballpark... 40K steps? 100k steps?

**Comment by dicroce** [comment]
Eventually I started seeing some bounding boxes. My loss is down to 4 or so, and I do get some bounding boxes on the VOC2007 test set. I'm not sure what is normal.

When you say the training will "settle" do you mean it will stop on its own? I know it will stop at 1000 epochs but will it stop on its own prior to that?

**Comment by onurbarut** [comment]
try different thresholds

--threshold 0.3

then you will see some bboxes.

**Comment by BenGHolmes** [comment]
Do I need to change the threshold when training or can I change it after training when running `flow --imgdir /PATH/TO/TEST/JPEGS --model cfg/yolo-voc.cfg --load -1 --gpu 1.0`
